pkg/resources/dynamic_table_test.go

```diff
@@ -0,0 +1,36 @@
+package resources_test
+
+import (
+	"context"
+	"fmt"
+
+	acc "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func init() {
+	resource.AddTestSweepers("snowflake_dynamic_table", &resource.Sweeper{
+		Name: "snowflake_dynamic_table",
+		F: func(profile string) error {
+			client, err := sdk.NewDefaultClient()
```

Comment:
And here this is initialized again.

Reply:
i think we only need a handful of sweeper functions. one for each account level object. then we can use "make sweep" to remove them. actually this code is not working right now, its just placeholder for "best practice". as i was reading through all the documentation on how to write acceptance tests for terraform, i saw this one and thought we should at least have a placeholder implementation